### ISSUE TITLE
sitemap: false doesn't work

### ISSUE THREAD

**Issue by MaCXyLo** [mmistakes/minimal-mistakes#991]
Hi,
i added the standard 404 file to my webpage:

```
title: "Page Not Found"
excerpt: "Page not found. Your pixels are in another canvas."
sitemap: false
permalink: /404.html
...
```

however, if i build the website the 404 page is included at the sitemap.
that's also the case on your example page:
https://mmistakes.github.io/minimal-mistakes/sitemap/

**Comment by mmistakes** [comment]
Check the **jekyll-sitemap** plugin's [documentation](https://github.com/jekyll/jekyll-sitemap#exclusions). You need to add `sitemap: false` to your post or pages you don't want included.

Closing as this isn't a theme issue.

**Comment by mmistakes** [comment]
Related issue: https://github.com/jekyll/jekyll-sitemap/pull/164

Apparently `sitemap: false` is no longer needed for 404.html pages

**Comment by coliff** [comment]
FYI; I found that `sitemap: false` doesn't work too - it was also showing CSS and JS files as well at sitemap.xml and robots.txt. Apparently it's fixed in v1.20 of the jekyll-sitemap plugin (https://github.com/jekyll/jekyll-sitemap/releases), but that's not available in GitHub Pages as of today.

I got around the limitation with a bit of jQuery on the sietmap.md page to hide items which works well.

```
$("a[href*='.css']").parents('.grid__item').remove();
$("a[href*='.js']").parents('.grid__item').remove();
$("a[href*='.xml']").parents('.grid__item').remove();
$("a[href*='.txt']").parents('.grid__item').remove();
```
